save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -124,10 +128,16 @@ class VulnerabilityService(
                     }
                 } ?: cb.and()
 
+                val tagsPredicate = tags.map { tag ->
+                    cb.like(root.get("tags"), tag)
```

Comment:
if we use like here, it can return all tags for substring too: `tags = "foo,bar"`, search by `"ar"` and get such vulnerability

Reply:
I know, and this seem to be a problem

Reply:
This problem seems to be fixed with a couple of new database tables